src/dune

```diff
@@ -1,6 +1,6 @@
-(library (name camlpstreams) (public_name camlpstreams)
+(library
+ (name camlp_streams)
+ (public_name camlp-streams)
  (modules Stream Genlex)
```

Comment:
AFAICT this line is not necessary.

Reply:
Which line?

Reply:
The `modules` line I think. bdb3abc4e7acce08df5a26ae70252133d7fc5751